glotaran/parameter/parameter_group.py

```diff
@@ -377,16 +379,48 @@ def update_parameter_expression(self):
                 parameter.value = value
 
     def markdown(self) -> str:
-        """Formats the :class:`ParameterGroup` as markdown string."""
-        t = "".join("  " for _ in range(self.get_nr_roots()))
-        s = ""
+        """Formats the :class:`ParameterGroup` as markdown string.
+
+        This is done by recursing the nested :class:`ParameterGroup` tree.
+        """
+        node_indentation = "  " * self.get_nr_roots()
+        return_string = ""
+        table_header = [
+            "_Label_",
+            "_Value_",
+            "_StdErr_",
+            "_Min_",
+            "_Max_",
+            "_Vary_",
+            "_Non-Negative_",
+            "_Expr_",
+        ]
         if self.label != "p":
```

Comment:
This line is a leftover from the limitations of lmfit's Parameters. Back then, the root group had the user invisible label `p`, no it is `None`. So the check should be `if label is not None`.